src/plugins/external_tool_support/src/blast/align_worker_subtasks/BlastAlignToReferenceTask.cpp

```diff
@@ -142,6 +142,7 @@ void BlastAlignToReferenceTask::prepare() {
     settings.gapOpenCost = 2;
     settings.gapExtendCost = 2;
     blastQuerySequenceIndexByReadIndex.resize(reads.size());
+    bool allReadsAreEmpty = true;
```

Comment:
You do not need this variable at all. You can check that `settings.querySequences` are empty or `settings.alphabet` is nullptr

Reply:
OK